terraform/ecs/linux/ec2_launch/daemon/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+variable "cwagent_image_repo" {
+  type    = string
+  default = "public.ecr.aws/cloudwatch-agent/cloudwatch-agent"
```

Comment:
So does this test against the public, globally deployed agent image or a local docker image build from the test pipeline? I tried looking for substitutions that point to our own testing image and didn't find any

Reply:
by default yeah looks like it. It should never be default though. This value is passed in from workflow: https://github.com/aws/amazon-cloudwatch-agent/pull/639/files#diff-9b9f9a38b889c9dcb55ba0330baa69c8fdd62ac1100b6eb59b6fc44008f5d320R727